templates/download/amd/_ubuntu_pro_section.html

```diff
@@ -0,0 +1,22 @@
+<div class="row p-strip u-no-padding--top">
+  <hr class="col-9" />
```

Comment:
You shouldn't need to use `col-9` here, as it is already in a `col-9` container. But when I remove it the alignment of the logo below breaks - not exactly sure why. But worth looking into, as `col-9` should be used like this and is prone to breakage if something changes.